src/richie/apps/search/indexers/courses.py

```diff
@@ -199,6 +234,24 @@ def get_data_for_es(cls, index, action):
                         language: course_run.extended_object.get_absolute_url(language)
                         for language in titles.keys()
                     },
+                    "categories": [
+                        str(pk) for pk in categories.values_list("pk", flat=True)
```

Comment:
Now that you call only, we use only, we should stick to the instance:
```suggestion
                        str(category.pk) for category in categories
```